packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -185,11 +185,22 @@ export class GuxAvatar {
     }
   }
 
+  private renderUcIntegrationsText(): JSX.Element | null {
+    if (['zoom', 'teams', '8x8'].includes(this.ucIntegration)) {
+      return (
+        <gux-screen-reader-beta>
+          {this.i18n(this.ucIntegration)}
+        </gux-screen-reader-beta>
+      ) as JSX.Element;
+    }
+  }
+
   private renderUcIntegrationBadge(): JSX.Element | null {
     if (this.ucIntegration && this.size === 'large') {
```

Comment:
Not critical but I think this function and the related child functions could be simplified/improved a bit. There are repeated checks related to `this.ucIntegration` and it is still technically be possible to return an empty `<div class="gux-avatar-integration-badge">` element.